src/main/resources/com/owlcyberdefense/mil-std-2045/xsd/milstd2045_application_header.dfdl.xsd

```diff
@@ -0,0 +1,628 @@
+              <element name="originator_address_group"
+                       minOccurs="0"
+                       dfdl:occursCountKind="expression"
+                       dfdl:occursCount="{ ../originator_address_group_PI/value }">
+                <complexType>
+                  <group ref="tns:urn_unit_name_group"/>
+                </complexType>
+              </element>
+              <sequence dfdl:hiddenGroupRef="msi:recipient_address_group_PI"/>
+              <element name="recipient_address_group" type="tns:address_group_type"
+                       minOccurs="0"
+                       dfdl:occursCountKind="expression"
+                       dfdl:occursCount="{ ../recipient_address_group_PI/value }"/>
+              <sequence dfdl:hiddenGroupRef="msi:information_address_group_PI"/>
+              <element name="information_address_group" type="tns:address_group_type"
+                       minOccurs="0"
+                       dfdl:occursCountKind="expression"
+                       dfdl:occursCount="{ ../information_address_group_PI/value }"/>
+              <sequence dfdl:hiddenGroupRef="msi:header_size_PI"/>
+              <element name="header_size"
+                       minOccurs="0"
+                       dfdl:occursCountKind="expression"
+                       dfdl:occursCount="{ ../header_size_PI/value }">
+                <complexType>
+                  <sequence>
+                    <element name="value" type="tns:tIntField" dfdl:length="16"
+                             dfdl:outputValueCalc="{ $tns:hdrLen }"/>
+                  </sequence>
+                </complexType>
+              </element>
+              <group ref="tns:futureUseGroup"/>
+              <!-- start message handling group -->
+              <element name="message_handling_group">
+                <complexType>
+                  <sequence>
+                    <element maxOccurs="unbounded" minOccurs="1" name="item"
+                             dfdl:occursCountKind="implicit">
+                      <complexType>
+                        <sequence>
+                          <sequence dfdl:hiddenGroupRef="msi:GRI_discriminator"/>
+                          <element name="umf">
+                            <complexType>
+                              <sequence>
+                                <element name="value" type="tns:umfEnum"/>
+                              </sequence>
+                            </complexType>
+                          </element>
+                          <sequence dfdl:hiddenGroupRef="msi:message_standard_version_PI"/>
+                          <element name="message_standard_version"
+                                   minOccurs="0"
+                                   dfdl:occursCountKind="expression"
+                                   dfdl:occursCount='{
+                                  if (../umf/value eq "OK_Binary_File" or
+                                      ../umf/value eq "OK_RDM" or
```

Comment:
I think this needs different definitions for C/DC1. ex: FWDMSG vs RDM, and NITFS being illegal in C, but not DC1...this could be covered by the "undefined" once the NITFS enum is updated for both C/DC1

Reply:
FWDMSG vs. RDM is just a renaming so adds nothing to validity checking, so definitely not creating a separate enum for that. 

NITFS for rev C isn't supposed to have a message_standard_version at all, as all values are undef. 
But it is not worth it to split the enums for detecting just that.